src/pages/Hosting/HostingStep/HostMap.js

```diff
@@ -0,0 +1,199 @@
+import React, { useState } from 'react';
+import { Map, MapMarker } from 'react-kakao-maps-sdk';
+import styled, { keyframes } from 'styled-components';
+
+const HostMap = ({ formData }) => {
+  const { kakao } = window;
+  const [inputAddr, setInputAddr] = useState('');
+  const [markers, setMarkers] = useState([]);
+  const [map, setMap] = useState();
+
+  const addrCheck = e => {
+    setInputAddr(e.target.value);
+  };
+
+  const enterSave = e => {
+    if (e.keyCode === 13) {
+      addressSave();
+    }
```

Comment:
```suggestion
    if (e.code === 'Enter') {
      addressSave();
    }
```

만약 Enter가 String으로 있는게 불편하시면 이렇게 관리하시면 됩니다!

```js
// src/lib/constants.js 
export const ActionKeys = { // enum처럼 사용하기 위해 파스칼케이스로 작성했습니다.
  Enter: 'Enter',
  ArrowUp: 'ArrowUp'
  .... etc
}
```

```js
    if (e.code === ActionKeys.Enter) {
      addressSave();
    }
```

Reply:
넵 String으로 처리해주었습니다!